includes/transformer/class-comment.php

```diff
@@ -107,8 +90,18 @@ protected function get_attributed_to() {
 	 * @return string The content.
 	 */
 	protected function get_content() {
-		$comment = $this->wp_object;
-		$content = $comment->comment_content;
+		$comment  = $this->wp_object;
+		$content  = $comment->comment_content;
+		$mentions = '';
+
+		foreach ( $this->extract_reply_context() as $acct => $url ) {
+			$mentions .= sprintf(
+				'<a class="u-mention mention" href="%s">%s</a> ',
```

Comment:
Unify HTML with mentions HTML

```suggestion
				'<a rel="mention" class="u-url mention" href="%s">%s</a> ',
```